opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -128,10 +127,13 @@ impl LoggerProvider {
                 Err(LogError::Other(format!("{errs:?}").into()))
             }
         } else {
+            let error = LogError::Other("logger provider already shut down".into());
+
             otel_warn!(
-                name: "logger_provider_already_shutdown"
+                name: "LoggerProvider.Shutdown.AlreadyShutdown",
+                error = error
```

Comment:
what is the need for passing error here? It does not give any additional information to user, right?

Reply:
It doesn't add any extra value. I tried for consistency by including both the error and the name everywhere in case of error/warning. But no strong opinion, we can remove if we agree.

Reply:
Have removed this now.